src/pages/CreateEventPage/components/FormFields/FormFields.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { InputText } from './styled'
+
+type CreateEventProps = {
+  id: string;
+  name: string;
+  label: string;
+  type?: string;
+  value: string;
+  onChange: Function;
+  options? : string[];
+}
+
+export function InputField({id, name, label, type, value, onChange}: CreateEventProps) {
+    return (
+      <InputText>
+        <label htmlFor={id}>{label}</label>
+          <input id={id} name={name} type={type} value={value} onChange={(e) => onChange(e.target.value)}/>
+      </InputText>
+    );
+  }
```

Comment:
What's up with the extra spaces here? The prettier plugin should fix these issues for you automatically.